runtime/ts/slot-dom-consumer.ts

```diff
@@ -12,6 +12,7 @@
 import {assert} from '../../platform/assert-web.js';
 import {SlotConsumer} from './slot-consumer.js';
 import Template from '../../shell/components/xen/xen-template.js';
+import IconStyles from '../../shell/components/icons.css.js';
```

Comment:
@raulverag @lindner as FYI on new runtime dependency on shell

Reply:
These files can be promoted above shell if that helps.